.gitignore

```diff
@@ -1,2 +1,4 @@
 node_modules
-bundle.js
\ No newline at end of file
+bundle.js
+product.csv
+review.csv
```

Comment:
Maybe dump these into a data folder and ignore the folder instead?

Reply:
Oh, it is dumped into a folder named CSV, im not quite sure why the folder disppeared